app/models/waste_carriers_engine/analytics/user_journey.rb

```diff
@@ -45,9 +45,12 @@ class UserJourney
       scope :date_range, lambda { |start_date, end_date|
         where(
           :created_at.gte => start_date.midnight,
-          :created_at.lt => end_date.midnight + 1.day,
-          :completed_at.gte => start_date.midnight,
-          :completed_at.lt => end_date.midnight + 1.day
+          :created_at.lt => end_date.midnight + 1.day
+        ).and(
+          :$or => [
+            { completed_at: nil },
```

Comment:
So we're including incomplete journeys in all cases?

Reply:
Yes, otherwise separate stats of "journeys started" and "journeys completed" would not be distinguished